script.js

```diff
@@ -0,0 +1,333 @@
+      isCapslock = true;
+    } else {
+      elem.innerHTML = '';
+      drawKeys(dataObject.lowerCaseEn, elem);
+      isCapslock = false;
+    }
+  });
+};
+
+const shiftHandler = (elem) => {
+  const shift = document.querySelectorAll('.shift');
+
+  shift.forEach((item) => {
+    item.addEventListener('mousedown', () => {
+      elem.innerHTML = '';
+      if (!isCapslock) drawKeys(dataObject.upperCaseEn, elem);
+      else drawKeys(dataObject.lowerCaseEn, elem);
+      elem.querySelectorAll('.shift')[0].classList.add('active');
+      elem.querySelectorAll('.shift')[1].classList.add('active');
+    });
+    item.addEventListener('mouseup', () => {
+      elem.innerHTML = '';
+      if (!isCapslock) drawKeys(dataObject.lowerCaseEn, elem);
+      else drawKeys(dataObject.upperCaseEn, elem);
+    });
+  });
+};
+
+const keyStyleHandler = (elem) => {
+  elem.addEventListener('mousedown', (event) => {
+    const { target } = event;
+    if (target.tagName === 'SPAN' || target.tagName === 'SECTION' || target.innerText === 'Shift' || target.innerText === 'CapsLock') return;
+    target.classList.add('active');
+  });
+  elem.addEventListener('mouseup', (event) => {
+    if (event.target.className === 'keyboard-styles' || event.target.innerText === 'Shift' || event.target.innerText === 'CapsLock') return;
+    event.target.classList.remove('active');
+  });
+};
+
+const printOnKeypress = () => {
+  document.addEventListener('keydown', (event) => {
+    event.preventDefault();
+    if (event.code === 'CapsLock') {
+      event.preventDefault();
+      if (isCapslock) {
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.lowerCaseEn, document.querySelector('.keyboard-styles'));
+        document.getElementById(event.code).classList.add('active');
+        isCapslock = false;
+      } else {
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.upperCaseEn, document.querySelector('.keyboard-styles'));
+        document.querySelectorAll('DIV').forEach((item) => {
+          item.classList.remove('active');
+        });
+        isCapslock = true;
+      }
+    }
+
+    if (event.code === 'ShiftLeft' || event.code === 'ShiftRight') {
+      // console.log(event.code);
+      event.preventDefault();
+      if (document.querySelector('.caps').classList.contains('active')) {
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.lowerCaseEn, document.querySelector('.keyboard-styles'));
+        document.querySelector('.caps').classList.add('active');
+      } else {
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.upperCaseEn, document.querySelector('.keyboard-styles'));
+      }
+    }
+
+    if (event.code === 'AltLeft' || event.code === 'AltRight' || event.code === 'Tab') {
+      event.preventDefault();
+    }
+
+    switch (document.getElementById(event.code).innerText) {
```

Comment:
Why do you use `document.getElementById(event.code).innerText`? It's better to add comment to such code

Reply:
fixed